parser/ast/stats.go

```diff
@@ -269,16 +269,17 @@ func (n *LoadStatsStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
-// LockStatsStmt is the statement node for lock table statistic
-type LockStatsStmt struct {
+// LockTableStatsStmt is the statement node for lock table statistic
+type LockTableStatsStmt struct {
 	stmtNode
 
-	Tables []*TableName
+	Tables         []*TableName
+	PartitionNames []model.CIStr
```

Comment:
I followed the AnalyzeTableStmt's style. @winoros @qw4990 Do you have any thoughts about this style issue?

Reply:
It's OK for me as long as there is no user using this experimental feature now.

Reply:
I prefer to reuse it to avoid any inconsistent problem, for example, `TableName.PartitionNames = {p1,p2}` but `LockTableStatsStmt.PartitionNames = {p3,p4}`.

Reply:
Got it. I will change it. Thanks!

Reply:
Updated!